static/sass/config/_fonts.scss

```diff
@@ -64,17 +64,18 @@ $code-font: (
   ),
+  'unicode-range': 'U+26',
```

Comment:
@mirisuzanne What does this do?

Reply:
Ah, I see: https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/unicode-range

Not relevant for Herman (at least until we implement more `@font` features).